src/pages/MyNotes/MyNotes.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@mui/material';
+
+import { NotesList } from 'pages/MyNotes';
```

Comment:
please create a folder called components inside MyNotes folder and move all components that used in MyNotes there
so your import will look like

`import { NotesList } from './components';`